firmware/application/apps/acars_app.cpp

```diff
@@ -31,73 +31,63 @@ using namespace acars;
 #include "string_format.hpp"
 #include "utility.hpp"
 
-void ACARSLogger::log_raw_data(const acars::Packet& packet, const uint32_t frequency) {
+void ACARSLogger::log_raw_data(const acars::Packet& packet,
+															 const uint32_t frequency) {
 	(void)frequency;
-	std::string entry { };	//= "Raw: F:" + to_string_dec_uint(frequency) + "Hz ";
+	std::string entry{};	//= "Raw: F:" + to_string_dec_uint(frequency) + "Hz ";
```

Comment:
`Cpp11BracedListStyle (true)` would leave the spacing there. Might make fewer edits that way.